naming_conventions.md

```diff
@@ -4,16 +4,16 @@ The page lists obligatory rules for our naming conventions. Please stick to it.
 
 ## 1 Project Name
 
-The project name <project-name> consists of <base-project>-<sub-project>, and only uses [a-z0-9] and dash -. No underline _.
+The project name [project-name] consists of [base-project]-[sub-project], and only uses [a-z0-9] and dash -. No underline _.
```

Comment:
Maybe we should use it consistently throughout the book? E.g. in *after_deployment_tasks.md:20* there is still <application> used.